### DISCUSSION TITLE
How do i set the new admin interface as standard interface?

### DISCUSSION THREAD

**Discussion by DJ-Key** [modoboa/modoboa#3047] [Help]
At the moment when i'm acessing mail.mydomain.xy it is the old interface and i have to go to mail.mydomain.xy/new-admin to get to the new pannel.
I want to configure the server in a way that i aceess the new interface directly from mail.mydomain.xy and don't have the old interface or that it is disabled so if anything goes wrong i can restore the old interface.

**Comment by isaaclepes** [comment]
You could use a redirect in the web server to redirect /dashboard to /new-admin (or whatever you set the new admin url as in your config).
